docs/Compatibility.md

```diff
@@ -12,10 +12,10 @@ The Ruby Datadog Trace library is open source. See the [dd-trace-rb][1] GitHub r
 |       |                            | 2.7     | Full                                 | Latest              |
 |       |                            | 2.6     | Full                                 | Latest              |
 |       |                            | 2.5     | Full                                 | Latest              |
-|       |                            | 2.4     | Full                                 | Latest              |
-|       |                            | 2.3     | Full                                 | Latest              |
-|       |                            | 2.2     | Full (except for Profiling)          | Latest              |
-|       |                            | 2.1     | Full (except for Profiling)          | Latest              |
+|       |                            | 2.4     | EOL since MM DDth, YYYY              | < 2.0.0             | [comment]: <> (# DEV-2.0: Update with release date)
```

Comment:
Hi @marcotc, looks like the dates are missing in these lines

Reply:
We don't know the dates yet :)
And won't know until closer to the release date.
But this document won't be published until then.